projects/kit/components/input-tag/input-tag.component.ts

```diff
@@ -85,7 +86,7 @@ const EVENT_Y_TO_X_COEFFICIENT = 3;
         HINT_CONTROLLER_PROVIDER,
     ],
 })
-export class TuiInputTagComponent
+export class TuiInputTagComponent<T = string>
```

Comment:
This would break build for everybody who use this class in code. I think we can just use `any` in `disabledItemHandler`.

Reply:
fixed